src/main/java/net/fabricmc/loader/launch/common/MappingConfiguration.java

```diff
@@ -51,7 +54,8 @@ public Mappings getMappings() {
 			}
 
 			if (mappings == null) {
-				mappings = MappingsProvider.createEmptyMappings();
+				LOGGER.warn("Could not find tiny mappings, mods using the mappings api will break.");
```

Comment:
If mappings are missing, that means the game isn't remapped either - I'm not sure we need to warn for this, as such games may exist (games which are deobfuscated from the factory).

Reply:
info then?

Reply:
I don't think we need to notify the user in this "stick" manner at all. Instead, perhaps an info message "Mappings present, namespaces: official, intermediary, named" (OR "Mappings not present!") would do.

Reply:
How about "mappings not present, mods using the mappings api _may_ not work"?

Reply:
I still feel this sends across an unreasonable message for non-Minecraft games. I'd agree to it for those game environments which use mappings by default - I think we could extend GameProvider with the necessary information.

Reply:
alright then